src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -69,13 +69,13 @@ public async Task<IActionResult> PushChangesAsync([FromBody]JToken jsonBody)
                 return BadRequest(validationMessage);
 
             string branch = jsonBody[$"branch"].ToString();
-            string file = jsonBody[$"file"].ToString();
-            string repoPath = jsonBody[$"repoPath"].ToString();
+            List<string> files = jsonBody[$"files"].Select(file => file.ToString()).ToList();
+            List<string> repoPaths = jsonBody[$"repoPaths"].Select(path => path.ToString()).ToList();
```

Comment:
Why do we need a list for repoPaths? List of files will go into the same repoPath, correct?

Reply:
the path includes the filename. so there will be "/detectorid/detectorid.csx" and "/detectorid/metadata.json"

Reply:
Got it. I thought its the parent path. Makes sense.